lib/db/mysql/patches/patch-005-006.sql

```diff
@@ -0,0 +1,20 @@
+-- Add refreshTokens
+
+CREATE TABLE refreshTokens (
+  token BINARY(32) PRIMARY KEY,
+  clientId BINARY(8) NOT NULL,
+  INDEX tokens_client_id(clientId),
+  FOREIGN KEY (clientId) REFERENCES clients(id) ON DELETE CASCADE,
+  userId BINARY(16) NOT NULL,
+  INDEX tokens_user_id(userId),
+  email VARCHAR(256) NOT NULL,
+  scope VARCHAR(256) NOT NULL,
+  createdAt TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
+) ENGINE=InnoDB CHARACTER SET utf8 COLLATE utf8_unicode_ci;
```

Comment:
I noticed that this table doesn't have a `type` column like the `tokens` table does.  Is it worth adding one?  It's not clear to me whether or not there could be different types of refresh token.

I guess we can always add one later if we introduce another token type.